### ISSUE TITLE
Can't join group via invite link.

### ISSUE THREAD

**Issue by s8321414** [Aseman-Land/Cutegram#179]
We may need to join group by via invite link, but I don't know why I can't do this on Cutegram.
I have tested on: Firefox, Chromium, Opera, Qupzilla, Rekonq, Konqueror on Chakra GNU/Linux.

**Comment by Techwolf** [comment]
Same here. Tried to join a group via invite link. Cutegram is being called by the browser, but nothing happens.

**Comment by s8321414** [comment]
yeah, browser calls xdg-open, then xdg-open calls cutegram to process tg:// but nothing happens in cutegram.

**Comment by realbardia** [comment]
Hi...
This feature is not supported by the cutegram currently. We'll add this in the next releases...

**Comment by s8321414** [comment]
@realbardia Thanks for your response! Glad to know this.

**Comment by Storm-Engineer** [comment]
I need telegram for the sole purpose of accessing a group and I cannot accept the invite... :/

You said in October that it will be in "next release". When that is expected to be?
